composer.json

```diff
@@ -49,6 +49,7 @@
         "aws/aws-sdk-php": "^2.7",
         "friendsofsymfony/rest-bundle": "^1.7.4 || ^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.1",
+        "matthiasnoback/symfony-dependency-injection-test": "^1.1",
```

Comment:
try `0.5 || ^1.0`maybe this fixes travis

Reply:
`"matthiasnoback/symfony-dependency-injection-test": "^0.7 || ^1.0",` because you use this method:
## v0.7.0

- Add `assertContainerBuilderNotHasService()`

you need both 0.x and 1.x to support --prefer-lowest and new packages

Reply:
@dmarkowicz I changed composer.json and pushed, should I do something else (don't know how to deal with --prefer-lowest thing)?

Reply:
nope, all travis green now :) If you want to test it locally before push, use 2 scenarios:
```bash
# for matthiasnoback/... package "^0.7 || ^1.0"
composer update # will install latest version: 1.1
make test
composer update --prefer-lowest # will install lowest version: 0.7
make test
```
If with --prefer-lowest tests not pass, you most likely use a feature that is not yet implemented. In this case need to read some changelogs and figure out which minimal version suits to you and meet other requirements like minimum php and symfony version supported by sonata.

Reply:
Thank you for the tip @dmarkowicz!